### PR TITLE
improve docs readability

### PR DESCRIPTION
**Pull request by DenisVuyka**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/Alfresco/alfresco-content-app/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] Application / Infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?

- Greatly improve readability, reduce amount of clicks to navigate content
- Concentrate content around three main areas: 
  * Home (Home + Features)
  * Getting Started (Build + Docker + Guides)
  * Get Help (FAQ + Get Help)
- better experience when navigating directly in the repo

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -17,35 +17,370 @@ with a simple and easy to use interface for working with files stored in the Alf
...
+The upload dialog persists on the screen and can be minimized; users are able to continue using the application whilst uploads are in progress
+and uploads can be canceled which will stop uploads in progress or permanently delete already completed uploads.
+
+![](images/uploader.png)
+
+#### Navigation
+
+The navigation links are configurable via the [app.config.json](https://github.com/Alfresco/alfresco-content-app/blob/master/src/app.config.json).
+Default configuration creates two sections.
+See [Navigation](#navigation) for more information about configuring the side navigation.
```
Comment (suzanadirla) [inline docs/README.md]: Denys, this link seems to be broken. Please check.

File: docs/README.md [hunk 2]
```diff
@@ -17,35 +17,370 @@ with a simple and easy to use interface for working with files stored in the Alf
...
+
+#### Versions tab
+
+The Versions tab displays info about the node's versions and allows users to [manage versions](/version-manager), according to their permissions. Only the file nodes have version data available.
+
+![](images/version-manager-tab.png)
+
+It uses the [VersionManagerComponent](https://alfresco.github.io/adf-component-catalog/components/VersionManagerComponent.html) from ADF framework.
+
+Managing versions of a file can be possible also by accessing the 'Manage Versions' option from the 'More actions' menu. For more info on manage versions, please check the [version manager](/version-manager) page.
```
Comment (suzanadirla) [inline docs/README.md]: the link to version manager is broken also

File: docs/README.md [hunk 3]
```diff
@@ -17,35 +17,370 @@ with a simple and easy to use interface for working with files stored in the Alf
...
+
+### Version Manager
+
+The versions of a file can be viewed & managed by using the [VersionManagerComponent](https://alfresco.github.io/adf-component-catalog/components/VersionManagerComponent.html).
+
+There are 2 ways users can access the Version Manager:
+
+1. From the 'Manage Versions' option of the 'More actions' menu (check [Actions and the Actions Toolbar](/doc-list?id=actions-and-the-actions-toolbar)):
```
Comment (suzanadirla) [inline docs/README.md]: Pls. check this broken link

File: docs/README.md (hunk 1 above)
Comment (DenisVuyka) [reply docs/README.md]: Navigation is a relative link within the page when docs are rendered. Should work fine when deployed but I will re-check.

Comment (suzanadirla) [reply docs/README.md]: ok, thanks

File: docs/README.md (hunk 2 above)
Comment (DenisVuyka) [reply docs/README.md]: thanks, fixed

**Pull request merged**